Scrum 149: BUG fix for Student and Equipment page

mengzhuou opened the pull request:
1. http://localhost:3000/Students
2. http://localhost:3000/Equipment
3. Removed Edit Button folder
4. Removed Edit Mode and Save button, the functionalities for delete and change role, etc, are all worked.

<img width="952" alt="image" src="https://github.com/user-attachments/assets/eb817f66-92ff-4211-bcdc-dcaa56c11b11">

Xisheng0725 reviewed (changes requested):
could u please using the button component in student portal to make sure abutton style is same with student portal

mengzhuou commented:
> could u please using the button component in student portal to make sure abutton style is same with student portal

1. Just replaced it with Button component
2. Adjusted the css for Table columns
3. Fixed the bug that the table result is now consistent with search result before and after switching role

denialpan reviewed (approved):
Looks good, you can merge this
